test/e2e/node/kubelet.go

```diff
@@ -310,9 +310,7 @@ var _ = SIGDescribe("kubelet", func() {
 		})
 
 		for _, itArg := range deleteTests {
-			name := fmt.Sprintf(
-				"kubelet should be able to delete %d pods per node in %v.", itArg.podsPerNode, itArg.timeout)
-			ginkgo.It(name, func() {
+			ginkgo.It("should let Kubelet delete x pods per node in n minutes", func() {
```

Comment:
The by message below should contain the information here.  
also you probably should capitalize the the letters that are variables 
[X] [N]

Reply:
The test has been made to use hardcoded values in 03cb4bf26f6, so the titled has subsequently been updated to not have `x` and `n` in it.